WordPress/Classes/ViewRelated/Site Creation/SiteCreationWizard.swift

```diff
@@ -1,12 +1,20 @@
 /// Coordinates the UI flow for creating a new site
 final class SiteCreationWizard: Wizard {
+    private lazy var content = {
+        WizardViewController()
+    }()
+
+    lazy var ui: UIViewController = {
```

Comment:
I like the name `content` above; I wonder if we could rename this property from `ui` to `contentViewController` in the `Wizard` protocol definition?

Reply:
I see your point. `contentViewController` sounds a bit redundant to me though (we'd have the type in the name and the actual type definition). I'll try `content`, just like that, and rename the private one.